src/list-grid.scss

```diff
@@ -0,0 +1,261 @@
+      }
+
+      @include fd-navigated() {
+        &::after {
+          content: "";
+          position: absolute;
+          top: 0;
+          right: 0;
+          height: 100%;
+          width: 0.1875rem;
+          display: block;
+          border-radius: 0 $fd-grid-list-border-radius $fd-grid-list-border-radius 0;
+          background-color: var(--sapList_SelectionBorderColor);
+        }
+      }
+    }
+  }
+
+  .#{$block}__toolbar {
+    @include fd-flex-vertical-center();
+
+    padding: 0 1rem;
+  }
+
+  .#{$block}__group-header {
+    @include fd-flex-vertical-center();
+    @include fd-ellipsis();
+
+    font-size: var(--sapFontHeader5Size);
+    padding-top: 0.75rem;
+    margin-top: 1rem;
+    height: 2.75rem;
+    color: var(--sapList_GroupHeaderTextColo);
+    font-weight: bold;
+    line-height: 2rem;
+
+    &:first-child {
+      margin-top: 0;
+    }
+  }
+
+  &__mode {
+    &--single-select-left,
+    &--multi-select {
+      .#{$item} {
+        .#{$block}__checkbox--label,
+        .#{$block}__radio--label {
+          @include fd-flex(column);
+
+          align-self: inherit;
+          color: var(--sapContent_LabelColor);
+        }
+      }
+    }
+
+    &--single-select-master,
+    &--multi-select,
+    &--single-select-left {
+      .#{$item} {
+        cursor: default;
+
+        @include fd-selected() {
+          box-shadow: $fd-grid-list-mode-box-shadow;
+          background-color: $fd-grid-list-item-selected-background-color;
+
+          &.#{$item}--link {
+            @include fd-hover() {
+              box-shadow: $fd-grid-list-mode-box-shadow;
+              background-color: $fd-grid-list-item-selected-background-color;
+            }
+
+            @include fd-active() {
+              background-color: $fd-grid-list-item-active-background-color;
+            }
+          }
+        }
+      }
+    }
+
+    &--single-select-master {
+      .#{$item} {
+        cursor: pointer;
+
+        @include fd-selected() {
+          cursor: default;
+        }
+      }
+    }
+  }
+
+  &__more {
+    cursor: pointer;
+    padding: 0.813rem;
+    background-color: var(--sapGroup_ContentBackground);
+    border-radius: $fd-grid-list-border-radius;
+    box-shadow: $fd-grid-list-item-box-shadow;
+    text-align: center;
+
+    .#{$more}--title {
+      font-weight: bold;
+    }
+
+    &--description {
+      margin: 0.813rem 0 0 0;
+    }
+
+    .#{$more}--title,
+    &--description {
+      color: var(--sapLink_Visited_Color);
+    }
+
+    @include fd-hover() {
+      background-color: $fd-grid-list-item-hover-background-color;
+    }
+
+    &:active {
+      background-color: $fd-grid-list-item-active-background-color;
+
+      .#{$more}--title,
+      .#{$more}--description {
+        color: $fd-grid-list-active-text-color;
+      }
+    }
+  }
+
+  &__footer {
+    color: var(--sapList_FooterTextColor);
+  }
+
+  @each $set-name, $colors in $fd-grid-list-highlight {
+    &__highlight--#{$set-name} {
```

Comment:
You are missing a `fd-grid-list__highlight` class that will bring the reset. You put there all the rules that are shared amongst all states (error, succes, etc). You then create a modifier class `fd-grid-list__highlight--error` that will overwrite the property specific for the base case. In this case it will be the color. The rest of the properties (position, top, left, etc. will go in the base class as they are shared). You need to have something like:
`<span class="fd-grid-list__highlight fd-grid-list__highlight--error">`

Reply:
fixed